### PR TITLE
Lab2 angular v14 upgrades

### PR DESCRIPTION
**Pull request by suhussai**
*Issue #, if available:*

*Description of changes:*
Lab2 angular v14 upgrades

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Lab2/client/Admin/angular.json [hunk 1]
```diff
@@ -16,74 +26,66 @@
...
-              "extractLicenses": true,
-              "vendorChunk": false,
-              "buildOptimizer": true,
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "10mb",
-                  "maximumError": "10mb"
+                  "maximumWarning": "2mb",
+                  "maximumError": "2mb"
```
Comment (ujwalbukka) [inline Lab2/client/Admin/angular.json:46]: change both to 10mb

File: Lab2/client/Admin/angular.json [hunk 2]
```diff
@@ -16,74 +26,66 @@
...
                 },
                 {
                   "type": "anyComponentStyle",
-                  "maximumWarning": "6kb",
+                  "maximumWarning": "2kb",
```
Comment (ujwalbukka) [inline Lab2/client/Admin/angular.json:50]: change to 6kb

File: Lab2/client/Landing/README.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
-# Landing
+# Dashboard
```
Comment (ujwalbukka) [inline Lab2/client/Landing/README.md:2]: Should be "Landing"

File: Lab2/client/Landing/angular.json [hunk 4]
```diff
@@ -44,60 +26,62 @@
...
-              "extractLicenses": true,
-              "vendorChunk": false,
-              "buildOptimizer": true,
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "10mb",
-                  "maximumError": "10mb"
+                  "maximumWarning": "2mb",
+                  "maximumError": "5mb"
```
Comment (ujwalbukka) [inline Lab2/client/Landing/angular.json:47]: change to 10mb

File: Lab2/scripts/deployment.sh [hunk 5]
```diff
@@ -38,53 +43,52 @@ fi
...
+    ex -sc '%s/s3_bucket = .*/s3_bucket = \"'$SAM_S3_BUCKET'\"/|x' ../../Lab6/server/tenant-samconfig.toml
   fi
 
   echo "Validating server code using pylint"
-  python3 -m pylint -E -d E0401 $(find . -iname "*.py" -not -path "./.aws-sam/*")
+  python3 -m pylint -E -d E0401 "$(find . -iname "*.py" -not -path "./.aws-sam/*")"
   if [[ $? -ne 0 ]]; then
-    echo "****ERROR: Please fix above code errors and then rerun script!!****"
-    exit 1
+    echo "****WARNING: Please fix code errors found in above files.****"
```
Comment (ujwalbukka) [inline Lab2/scripts/deployment.sh:79]: As discussed update this

File: Solution/Lab2/client/Admin/angular.json [hunk 6]
```diff
@@ -16,74 +26,66 @@
...
-              "extractLicenses": true,
-              "vendorChunk": false,
-              "buildOptimizer": true,
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "10mb",
-                  "maximumError": "10mb"
+                  "maximumWarning": "2mb",
+                  "maximumError": "2mb"
```
Comment (ujwalbukka) [inline Solution/Lab2/client/Admin/angular.json:46]: change to 10mb

File: Solution/Lab2/client/Admin/angular.json [hunk 7]
```diff
@@ -16,74 +26,66 @@
...
                 },
                 {
                   "type": "anyComponentStyle",
-                  "maximumWarning": "6kb",
+                  "maximumWarning": "2kb",
```
Comment (ujwalbukka) [inline Solution/Lab2/client/Admin/angular.json:50]: change to 6kb

File: Solution/Lab2/client/Landing/README.md [hunk 8]
```diff
@@ -1,27 +1,27 @@
-# Landing
+# Dashboard
```
Comment (ujwalbukka) [inline Solution/Lab2/client/Landing/README.md:2]: "Landing"

File: Solution/Lab2/scripts/deployment.sh [hunk 9]
```diff
@@ -38,53 +43,52 @@ fi
...
+    ex -sc '%s/s3_bucket = .*/s3_bucket = \"'$SAM_S3_BUCKET'\"/|x' ../../Lab6/server/tenant-samconfig.toml
   fi
 
   echo "Validating server code using pylint"
-  python3 -m pylint -E -d E0401 $(find . -iname "*.py" -not -path "./.aws-sam/*")
+  python3 -m pylint -E -d E0401 "$(find . -iname "*.py" -not -path "./.aws-sam/*")"
   if [[ $? -ne 0 ]]; then
-    echo "****ERROR: Please fix above code errors and then rerun script!!****"
-    exit 1
+    echo "****WARNING: Please fix code errors found in above files.****"
```
Comment (ujwalbukka) [inline Solution/Lab2/scripts/deployment.sh:79]: As discussed update this

**Comment by suhussai** [comment]
Replaced by https://github.com/aws-samples/aws-serverless-saas-workshop/pull/24

**Pull request closed without merging**